src/components/Table/TableRow/TableRow.tsx

```diff
@@ -1,16 +1,46 @@
-import React, { FC } from "react";
-import { VibeComponentProps } from "../../../types";
+import React, { FC, useCallback, useRef } from "react";
 import styles from "./TableRow.module.scss";
+import { VibeComponentProps } from "../../../types";
 import { ITableCellProps } from "../TableCell/TableCell";
+import { useKeyEvent } from "../../../hooks";
+import { SELECTION_KEYS } from "../../../constants";
+import { NOOP } from "../../../utils/function-utils";
 
 export interface ITableRowProps extends VibeComponentProps {
+  highlight?: boolean;
+  onClick?: (event: React.MouseEvent | React.KeyboardEvent) => void;
   children?: React.ReactElement<ITableCellProps> | React.ReactElement<ITableCellProps>[];
   style?: React.CSSProperties;
 }
 
-const TableRow: FC<ITableRowProps> = ({ children, style = {} }) => {
+const TableRow: FC<ITableRowProps> = ({ highlight = false, onClick = NOOP, children, style = {} }) => {
```

Comment:
This should be fine as well without the default values
```suggestion
const TableRow: FC<ITableRowProps> = ({ highlight, onClick, children, style }) => {
```

Reply:
having no default for highlight would force me do two things:
```aria-selected={highlight || false}```
for onclick it makes sense, I'll add a check before calling the method itself

Reply:
cool, so whatever you prefer